vamb/h_loss.py

```diff
@@ -0,0 +1,820 @@
+"""Hierarchical loss for the labels suggested in https://arxiv.org/abs/2210.10929"""
+
+
+__cmd_doc__ = """Hierarchical loss for the labels"""
+
+import torch as _torch
+from torch import nn as _nn
+from torch.utils.data.dataset import TensorDataset as _TensorDataset
+import torch.nn.functional as F
+from typing import Optional, IO, Union
+from pathlib import Path
+from torch.utils.data.dataset import TensorDataset as _TensorDataset
+from torch.utils.data import DataLoader as _DataLoader
+from torch.optim import Adam as _Adam
+from torch import Tensor
+_torch.manual_seed(0)
```

Comment:
There is a `--seed` option for Vamb - if possible, just get it from there instead of seeding when the module is loaded.